src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/entities/BaseCommandArguments.java

```diff
@@ -170,11 +170,12 @@ public void setLogger(Logger logger) {
 
     @Override
     public String toString() {
+        String tokenHint = "..."+token.substring(token.length() - 10);
```

Comment:
Please extract the logic to prettify a token to a utility and use it in both places.